src/image/PhotoCalib.cc

```diff
@@ -45,38 +46,41 @@ namespace image {
 
 namespace {
 
-double toMaggies(double instFlux, double scale) { return instFlux * scale; }
+double toNanojansky(double instFlux, double scale) { return instFlux * scale; }
 
-double toMagnitude(double instFlux, double scale) { return -2.5 * log10(instFlux * scale); }
+double toMagnitude(double instFlux, double scale) { return utils::nanojanskyToABMagnitude(instFlux * scale); }
 
-double fromMagnitude(double magnitude, double scale) { return pow(10, magnitude / -2.5) / scale; }
+double fromMagnitudeToInstFlux(double magnitude, double scale) {
+    return utils::ABMagnitudeToNanojansky(magnitude) / scale;
+}
```

Comment:
A quick look at the code, I see `ABMagnitudeToNanojansky` but this is converting to `instFlux` not `nJy`!  Then I notice the scale.  I think that a code comment here would be helpful for future readers.  (As above with the `toMagnitude`).  But why is that not `toMagnitudeFromInstFlux` for symmetry?

Reply:
I've renamed it and add a hopefully clarifying comment.